### ISSUE TITLE
[FEATURE REQUEST] Filter for DEMO's

### ISSUE THREAD

**Issue by BigBodZod** [IsThereAnyDeal/AugmentedSteam#1420]
### Current Behavior (if any)
You can no longer tag/add DEMO's to your wishlist, you can tag the BASE game however but when doing a search sorted by New Releases for instance, the BASE game will highlight, if you have chosen to highlight items on your wishlist, but the DEMO will not tag.

### Desired Behavior
Would love to be able to HIDE DEMO's since I can no longer tag them and use the HIDE items on my Wishlist function.

### Motivation / Use Case for Changing the Behavior
Basically I wishlist EVERYTHING I don't have in my Steam library so adding items to the wishlist is so I know what I don't already have in my library.

**Comment by BigBodZod** [comment]
When I do a New Releases Search, I like to search by ALL types so I can add to my Wishlist.

Since DEMO's can no longer be tagged to be placed onto my Wishlist I need a way to HIDE them on the New Releases search pages.

Yes I can use the type filter and search by only GAMES or by only DLC etc and yes this will hide DEMO's

**Comment by prrrszalony** [comment]
I would like something similar - to be able to filter my wishlist and see all the games that have demos.

**Comment by candela97** [comment]
@prrrszalony that's tracked at #447
